packages/flutter/lib/src/cassowary/result.dart

```diff
@@ -17,24 +17,38 @@ class Result {
   /// Whether this [Result] represents an error (true) or not (false).
   final bool error;
 
+  /// The result when the operation was successful.
   static const Result success =
       const Result._('Success', isError: false);
 
+  /// The result when the constraint could not be added to the solver because it
+  /// was already present in the solver.
   static const Result duplicateConstraint =
       const Result._('Duplicate constraint');
 
+  /// The result when the constraint could not be added to the solver because it
+  /// was unsatisfiable. It is recommeded that the priority of the constraint be
```

Comment:
Instead of "It is recommeded that the priority of the constraint be lowered and the operation tried again.", I would suggest "If you get this result, try lowering the priority of the constraint" or some such.